onnxruntime/core/platform/posix/env.cc

```diff
@@ -123,7 +128,9 @@ class PosixEnv : public Env {
     auto length_remain = len;
     do {
       size_t bytes_to_read = length_remain;
-      ssize_t bytes_readed = offset > 0 ? pread(fd, wptr, bytes_to_read, offset) : read(fd, wptr, bytes_to_read);
+      ssize_t bytes_readed;
```

Comment:
bytes_readed should be just 'bytes_read' (read is the past tense of read, but it's pronounced differently...)